backend/src/packages/journal-entries/journal-entry.controller.ts

```diff
@@ -77,6 +77,9 @@ class JournalEntryController extends BaseController {
     this.addRoute({
       path: JournalApiPath.ROOT,
       method: 'POST',
+      validation: {
+        body: createJournalEntryValidationSchema,
+      },
```

Comment:
It is unnecessary here, as in create I just send the default payload, so user can't send anything
```suggestion
      
```

Reply:
@OlenaYatseniuk it's always good to add a validation in backend, because the user might use Swagger or Postman to make requests

Reply:
@fshabanov ok, thanks for the clarification

Reply:
Yeah, I was also taking into consideration a Postman when was adding it